.github/workflows/main.yml

```diff
@@ -79,4 +86,33 @@ jobs:
       with:
         push: true
         context: ./src
-        tags: ${{ secrets.DOCKERHUB_USERNAME }}/${{ env.DockerImageName }}:${{ github.sha }}
\ No newline at end of file
+        tags: ${{ secrets.DOCKERHUB_USERNAME }}/${{ env.DockerImageName }}:${{ github.sha }}
+
+  helm-ci:
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_name == 'main' }}
```

Comment:
The 'helm-ci' job is conditioned to run only on the 'main' branch. If you want to perform Helm operations for other branches, consider removing or modifying this condition.